File: 251117/Date to Date/date-to-date.py
m1, d1, m2, d2 = map(int, input().split())

month_days = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
result = 0
month_idx = m1

while True:
    if m1 == m2 and d1 == d2:
        break
    
    if d1 == month_days[month_idx]:
        m1 += 1
        d1 = 0
        result += 1
        month_idx += 1
    
    d1 += 1
    result += 1

print(result)

File: 260304/Date to Date/date-to-date.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const [m1, d1, m2, d2] = input[0].split(' ').map(Number);

const dates = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];

const calculateDates = (month, day) => {
    startMonth = 1;
    startDay = 1;
    totalDay = 0;

    while (true) {
        if (startMonth === month && startDay === day) {
            break;
        }

        totalDay += 1;
        startDay += 1;

        if (startDay > dates[startMonth]) {
            startMonth += 1;
            startDay = 1;
        }
    }
    return totalDay;
}

const daySum1 = calculateDates(m1, d1);
const daySum2 = calculateDates(m2, d2);

(daySum1 > daySum2) ? console.log(daySum1 - daySum2 + 1) : console.log(daySum2 - daySum1 + 1);
